### DISCUSSION TITLE
Copyright question

### DISCUSSION THREAD

**Discussion by swhansw** [obsproject/obs-studio#4782] [Development Questions] [answered]
I am trying to make a guide video for teachers how to use OBS and distribute it to all the schools in my country(I am doing this as a government project)
OBS interface screenshots and the likes will be used in the video.
Would there be any copyright issues I need to be aware of?
I know that OBS is not copyrighted. I just want to make sure.

**Comment by ElectronicWar** [comment, marked as answer]
Yes, this is safe. You can show OBS in your videos without any issues.

Just do not use the OBS logo to market your video as it is trademarked (showing it in your video is no problem as it belongs to the app) or promote it in such way as if your video is affiliated with the OBS Project in any way.

**Comment by skeletonbow** [comment]
> I know that OBS is not copyrighted. I just want to make sure.

It's important to note that OBS is copyrighted.  It is available under the terms of the GNU General Public License v.2 which you can view here:  https://www.gnu.org/licenses/old-licenses/gpl-2.0.html

All creative works (music, video, art, software, etc.) are copyrighted by default upon creation, and subject to the terms of whatever license the copyright holder chooses to make them available under.  It's important to understand that open source software is still copyrighted software available under a license with specific terms.  Usage of OBS in your video is of course perfectly ok.

Thought I'd mention this since copyright law is frequently misunderstood.

**Comment by swhansw** [comment]
Thank you very much for your kind explanation.
I will keep it in mind.
​
Thank you again.
​
Regards​
​
-----Original Message-----
